product-service/handler.ts

```diff
@@ -1 +1,24 @@
-export { getProductsList, getProductById } from '@/functions';
+import { DynamoDB } from 'aws-sdk';
+import * as handlers from '@/functions';
+import { ProductProvider } from '@/providers/productProvider';
+
+console.log(process.env);
+
+const { IS_OFFLINE, PRODUCTS_TABLE_NAME = '' } = process.env;
+
+const dbOfflineOptions = {
+  region: 'localhost',
+  endpoint: 'http://localhost:8000',
+  accessKeyId: 'DEFAULT_ACCESS_KEY',
+  secretAccessKey: 'DEFAULT_SECRET',
+};
+
+const dbClient = IS_OFFLINE
```

Comment:
suggestion: the better check  `process.env?.IS_LOCAL === 'true' || process.env?.IS_OFFLINE === 'true'`